main.go

```diff
@@ -39,6 +39,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	// get configs from meshsync crd if available
+	crdConfigs, err := config.GetMeshsyncCRDConfigs()
+
+	if err != nil {
+		// no configs found from meshsync CRD log warning
```

Comment:
I am not sure the log level should be warning level, please get confirm from @leecalcote @theBeginner86. I assume that if we cannot get configs, it should be the Info level? Because we have the default configuraiton.

Reply:
The choice of using a warning log is because the desired state of configuration for Meshsync from now and in future lies is utilizing the crd as per conversations with @leecalcote  and @theBeginner86  so this is an extra speed bump to notify one that the desired approach has not been utilized